spec/workers/gitter_notificator_spec.rb

```diff
@@ -0,0 +1,110 @@
+      allow(Settings).to receive(:travis_monitor).and_return(item_travis_monitor)
+
+      subject.send(:init)
+
+      expect(subject.instance_variable_get(:@gitter_token)).to eq(token)
+    end
+
+    it "should call check with proper parameters" do
+      allow(item_travis_monitor).to receive(:branches).and_return([item_branch])
+      allow(Settings).to receive(:travis_monitor).and_return(item_travis_monitor)
+
+      expect(subject).to receive(:check).with(item_branch.repo, item_branch.branch, item_branch.gitter_room)
+
+      subject.send(:init)
+    end
+
+    it "should call check 3 times" do
+      allow(item_travis_monitor).to receive(:branches).and_return([item_branch, item_branch, item_branch])
+      allow(Settings).to receive(:travis_monitor).and_return(item_travis_monitor)
+
+      expect(subject).to receive(:check).exactly(3).times
+
+      subject.send(:init)
+    end
+  end
+
+  describe "#check" do
+    it "should send a broken branch message" do
+      allow(subject).to receive(:latest_builds).and_return(%w(passed failed))
+      allow(subject).to receive(:gitter_send)
+
+      expect(subject).to receive(:gitter_send).with(room, ":sos: :warning: \"#{branch}\" in \"#{repository}\" is broken :bangbang: :boom:")
+
+      subject.send(:check, repository, branch, room)
+    end
+
+    it "should send a fixed branch message" do
+      allow(subject).to receive(:latest_builds).and_return(%w(failed passed))
+      allow(subject).to receive(:gitter_send)
+
+      expect(subject).to receive(:gitter_send).with(room, ":white_check_mark: Broken branch has been fixed :green_heart:")
+
+      subject.send(:check, repository, branch, room)
+    end
+
+    it "should not send a message on passing builds" do
+      allow(subject).to receive(:latest_builds).and_return(%w(passed passed))
+      allow(subject).to receive(:gitter_send)
+
+      expect(subject).not_to receive(:gitter_send)
+
+      subject.send(:check, repository, branch, room)
+    end
+
+    it "should not send a message on failing builds" do
+      allow(subject).to receive(:latest_builds).and_return(%w(failed failed))
+      allow(subject).to receive(:gitter_send)
+
+      expect(subject).not_to receive(:gitter_send)
+
+      subject.send(:check, repository, branch, room)
+    end
+  end
+
+  describe "#latest_builds" do
+    let(:first_expected_build) { double("Build", :state => "passed", :branch_info => branch) }
+    let(:second_expected_build) { double("Build", :state => "failed", :branch_info => branch) }
+
+    let(:valid_unexpected_build) { double("Build", :state => "state", :branch_info => branch) }
+    let(:invalid_unexpected_build) { double("Build", :state => "state", :branch_info => "wrong branch") }
+
+    let(:travis_builds) { [first_expected_build, invalid_unexpected_build, second_expected_build, valid_unexpected_build] }
+    let(:travis_repository) { double("Travis::Repository", :builds => travis_builds) }
```

Comment:
Why do you need the `travis_build` variable? Use just a single multiline let block.